react/w-books/src/app/index.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 
+import BooksApp from '~components/BooksApp';
+
 import '../scss/application.scss';
-import { AppRouter } from './routers/AppRouter';
 
 function App() {
-  return <AppRouter />;
+  return <BooksApp />;
```

Comment:
I don't understand why you deleted this AppRouter here, and move it to another component that returns it to itself